plenum/common/signer_did.py

```diff
@@ -33,11 +47,22 @@ def identifier(self) -> Identifier:
 
     @property
     def verkey(self) -> str:
+        if self._verkey is None:
+            return None
+
         if self.abbreviated:
             return self.abbr_prfx + self._verkey
         else:
             return self._verkey
 
+    def full_verkey(self):
+        if self.abbreviated:
```

Comment:
This method will throw AttributeError for `abbreviated` if `DidIdentity` was created without passing `verkey` or `rawVerkey`